iina/FileGroup.swift

```diff
@@ -17,7 +17,6 @@ class FileInfo: Hashable {
   var filename: String
   var ext: String
   var nameInSeries: String?
-  var characters: [Character]
```

Comment:
Arrays are COW. Since no modifications are taking place no copying should be done.